nat-lab/tests/uniffi/libtelio_remote.py

```diff
@@ -31,6 +32,34 @@ def wrap(*args, **kwargs):
     return wrap
 
 
+# This function is meant to be a temporary way to deal with
+# structured events until natlab has been migrated to use
+# the types provided by the generated bindings
+def serialize_event(event: libtelio.Event) -> str:
+    def extract_value(original) -> str:
+        return str(original).lower().rsplit(".", maxsplit=1)[-1]
+
+    event_dict = {}
+    body = event.body.__dict__
+    body["public_key"] = base64.b64encode(body["public_key"]).decode("utf-8")
+    if event.is_relay():
+        event_dict["type"] = "relay"
+        body["conn_state"] = extract_value(body["conn_state"])
+    elif event.is_node():
+        event_dict["type"] = "node"
+        body["state"] = extract_value(body["state"])
+        if body["link_state"] is not None:
+            body["link_state"] = extract_value(body["link_state"])
+        body["path"] = extract_value(body["path"])
+    elif event.is_error():
+        event_dict["type"] = "error"
```

Comment:
Shouldn't it contain error level as well?

Reply:
That's very correct! I never came across an error event while implementing this so I missed that, but it's added now

Reply:
Or maybe I didn't quite understand what you meant? It made me realize I didn't handle error level and error code in error events, but is that what you meant?

Reply:
Yeah, it was exactly about that